docs/api/spec/paths/violations.yaml

```diff
@@ -0,0 +1,106 @@
+/violations:
+  get:
+    summary: Returns a list of violations
+    description: The list of vilations will be limited to 100 and a cursor will be returned for iterating on other violations. Iteration is only needed if the nextCursor is not an empty value.
+    parameters:
+      - $ref: '#/components/parameters/cursor'
+      - $ref: '#/components/parameters/additional_fields'
+    responses:
+      '200':
+        description: List of violations
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ViolationList"
+  post:
+    summary: Creates a violation
+    requestBody:
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/Violation"
+    responses:
+      '201':
+        $ref: "#/components/responses/Created"
+      '400':
+        $ref: "#/components/responses/BadRequest"
+      '409':
+        $ref: "#/components/responses/Duplicate"
+      '422':
+        $ref: "#/components/responses/UnprocessableEntity"
+
+'/violation/mac/{mac}':
+  get:
+    summary: Allows to search through violations by MAC address
+    description: The list of violations will be limited to 100 and a cursor will be returned for iterating on other violations. Iteration is only needed if the nextCursor is not an empty value.
+    parameters:
+      - $ref: '#/components/parameters/search'
+      - $ref: '#/components/parameters/logic_operator'
```

Comment:
I don't think we need the search payload and logic operator since from what I can see the only thing it searches with is the MAC in the URL